<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>capy::ui::WelcomeScreen</class>
    <widget class="QMainWindow" name="capy::ui::WelcomeScreen">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>700</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Welcome to CapyArtStudio</string>
        </property>
        <property name="windowIcon">
            <iconset resource="../../resources/resources.qrc">
                <normaloff>:/icon.png</normaloff>:/icon.png
            </iconset>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                    <widget class="QFrame" name="detailsFrame">
                        <property name="frameShape">
                            <enum>QFrame::NoFrame</enum>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                                <widget class="QLabel" name="label">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>128</width>
                                            <height>128</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="pixmap">
                                        <pixmap resource="../../resources/resources.qrc">:/icon.png</pixmap>
                                    </property>
                                    <property name="scaledContents">
                                        <bool>true</bool>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="logoLabel">
                                    <property name="font">
                                        <font>
                                            <pointsize>26</pointsize>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Capy Art Studio</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="versionLabel">
                                    <property name="font">
                                        <font>
                                            <pointsize>14</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Version 1.0.0</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QPushButton" name="createProjectButton">
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Create New Project</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="openProjectButton">
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Open Project</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="importProjectButton">
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Import File as Project</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="continueButton">
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Continue without Project</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="settingsButton">
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Settings</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="exitButton">
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Exit</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </item>
                <item>
                    <widget class="QFrame" name="sidebarFrame">
                        <property name="frameShape">
                            <enum>QFrame::NoFrame</enum>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_2">
                            <property name="leftMargin">
                                <number>0</number>
                            </property>
                            <property name="rightMargin">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QLabel" name="sidebarLabel">
                                    <property name="font">
                                        <font>
                                            <pointsize>16</pointsize>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Projects</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="horizontalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>0</width>
                                            <height>0</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QScrollArea" name="scrollArea">
                                    <property name="widgetResizable">
                                        <bool>true</bool>
                                    </property>
                                    <widget class="QWidget" name="scrollAreaWidgetContents">
                                        <property name="geometry">
                                            <rect>
                                                <x>0</x>
                                                <y>0</y>
                                                <width>439</width>
                                                <height>513</height>
                                            </rect>
                                        </property>
                                    </widget>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </item>
            </layout>
        </widget>
    </widget>
    <layoutdefault spacing="6" margin="11"/>
    <resources>
        <include location="../../resources/resources.qrc"/>
    </resources>
    <connections/>
</ui>
